src/MSALWrapper/PCACache.cs

```diff
@@ -37,20 +38,26 @@ internal class PCACache
         /// </summary>
         /// <param name="logger">The logger.</param>
         /// <param name="tenantId">The tenant id.</param>
-        /// <param name="cacheFilePath">The cache file name.</param>
-        /// <param name="osxKeyChainSuffix">The osx key chain suffix.</param>
-        internal PCACache(ILogger logger, Guid tenantId, string cacheFilePath, string osxKeyChainSuffix = null)
+        internal PCACache(ILogger logger, Guid tenantId)
         {
             this.logger = logger;
-            this.osxKeyChainSuffix = string.IsNullOrWhiteSpace(osxKeyChainSuffix) ? $"{tenantId}" : $"{osxKeyChainSuffix}.{tenantId}";
+            this.osxKeyChainSuffix = $"{OSXKeyChainCategory}.{tenantId}";
 
-            if (string.IsNullOrWhiteSpace(cacheFilePath))
+            this.cacheFileName = $"msal_{tenantId}.cache";
+            this.cacheDir = this.CacheServiceFoler;
+        }
+
+        /// <summary>
+        /// Gets the absolute path of the cache folder. Only available on Windows.
+        /// </summary>
+        public string CacheServiceFoler

```

Comment:
Is there any need for this to be public?

Reply:
Changed to private